01 - Intro/part7_scope.c

```diff
@@ -31,6 +31,14 @@ int main(void)
 
     /* (2) Scope */
 
+    uint32_t local_var = 6;
+    printf("Local Var: %x\n", local_var);
+    {
+        uint32_t local_var = 5;
```

Comment:
Just keep in mind this is not ANSI-C and sometimes prohibited in coding conventions